test/dataloader_test.exs

```diff
@@ -232,6 +242,26 @@ defmodule DataloaderTest do
       assert result == {:ok, [id: "ben", username: "Ben Wilson"]}
     end
 
+    test "get/4 returns an {:ok, value} tuple when data is value tuple", %{loader: loader} do
+      result =
+        loader
+        |> Dataloader.load(:test, :results, :succes_data)
+        |> Dataloader.run()
+        |> Dataloader.get(:test, :results, :succes_data)
+
+      assert result == {:ok, :value}
+    end
+
+    test "get/4 returns an {:ok, value} tuple when data is error tuple", %{loader: loader} do
```

Comment:
```suggestion
    test "get/4 returns an {:error, value} tuple when data is error tuple", %{loader: loader} do
```

Reply:
@mbuhot, could you approve?

Reply:
@altuntasfatih: This test looks like it has the wrong name, I think @mbuhot is right.

Reply:
yeah @benwilson512, I agree it is wrong. 
Actually, I had changed it from here(sending the suggested commit) then I overrode it from the local commit mistakenly.

Reply:
I have just amended it.